packages/zero-solid/vitest.config.ts

```diff
@@ -1 +1,5 @@
-export {default} from '../shared/src/tool/vitest-config.ts';
+import solid from 'vite-plugin-solid';
+import {defineConfig, mergeConfig} from 'vitest/config';
+import config from '../shared/src/tool/vitest-config.ts';
+
+export default mergeConfig(config, defineConfig({plugins: [solid()]}));
```

Comment:
Is this needed? Maybe we were not using jsx in the tests before?

Reply:
exactly, added because we now are using jsx in the tests.